packages/angular/src/utils/testing.ts

```diff
@@ -50,6 +53,9 @@ export function runMigration<SchemaOptions = any>(
     .toPromise();
 }
 
+/**
+ * @deprecated will be removed in v13.
+ */
 export function runExternalSchematic<SchemaOptions = any>(
```

Comment:
What is the alternative?

Reply:
None, as the PR description says, we're not going to support them moving forward, we support Nx DevKit utilities. These were always intended to be private anyway.

Reply:
No problem 👍